view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -458,7 +458,7 @@ define(
                     mount(actionNode);
                 }
             },
-            handleOnSubmit: function(state, component) {
+            handleOnSubmit: async function(state, component) {
```

Comment:
Do we need to update other usages of handleOnSubmit because of this change?

Reply:
We used it [here](https://github.com/Adyen/adyen-magento2/pull/1435/files#diff-7de10d96985d87af113c028849c660af1c3365ec9a89f8a63af784635d16bea2R755) with the `onSubmit` for amazonpay.